build.sbt

```diff
@@ -1717,9 +1717,15 @@ lazy val prepareItLibs = taskKey[Unit]("Prepare jar libraries needed for integra
 lazy val restmodel = (project in file("designer/restmodel"))
   .settings(commonSettings)
   .settings(
-    name := "nussknacker-restmodel"
+    name := "nussknacker-restmodel",
+    libraryDependencies ++= {
+      Seq(
+        "com.softwaremill.sttp.tapir" %% "tapir-core"       % tapirV,
+        "com.softwaremill.sttp.tapir" %% "tapir-json-circe" % tapirV
+      )
+    }
   )
-  .dependsOn(extensionsApi, commonApi % "test->test", testUtils % "test")
+  .dependsOn(extensionsApi, commonApi % "test->test", testUtils % "test", security)
```

Comment:
Please look at [the diagrage](https://nussknacker.io/documentation/docs/developers_guide/Basics/) and see the API-related modules.
IMO we don't want to pull in the `security` module. I thought we could move only some classes (related to the model) to the `restmodel` module.